### PR TITLE
docs: add initial table for added, updated, unchanged

### PR DESCRIPTION
**Pull request by ther3sa**
## Description
PR to collaboratively work on the Changelog for R24.12

- [ ] Changed components
- [ ] Unchanged/untetsted components
- [ ] incompatible
- [ ] known knowns 
- [ ] runtimes

## [24.12] - 2024-12-02

Implemented features can be found [here](https://github.com/orgs/eclipse-tractusx/projects/26/views/33?filterQuery=milestone%3A%2224.12%22+-label%3Ametadata+is%3Aclosed++status%3ADone)

### Updated

| Component | Helm Chart (s) | App-/KIT Version (s) |
| :-------- | :-------------: | :------------------: |
| [BPDM KIT](https://eclipse-tractusx.github.io/docs-kits/category/business-partner-kit) | n/a | [7.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Changelog) |
| [Circularity KIT](https://eclipse-tractusx.github.io/docs-kits/kits/Circularity_KIT/page-adoption-view) | n/a | [1.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/Data%20Chain%20Kit/data%20chain%20kit%20changelog) |
| [DCM Kit](https://eclipse-tractusx.github.io/docs-kits/kits/Demand%20and%20Capacity%20Management%20Kit/adoption-view/overview) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/Demand%20and%20Capacity%20Management%20Kit/changelog) |
| [Digital Twin Registry](https://github.com/eclipse-tractusx/sldt-digital-twin-registry) | digital-twin-registry: [0.6.2](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/digital-twin-registry-0.6.2) | [0.6.2](https://github.com/eclipse-tractusx/sldt-digital-twin-registry/releases/tag/v0.6.2) |
| [Discovery Finder](https://github.com/eclipse-tractusx/sldt-discovery-finder) | discoveryfinder: [0.4.2](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/discoveryfinder-0.4.2) | [0.5.2](https://github.com/eclipse-tractusx/sldt-discovery-finder/releases/tag/v0.5.2) |
| [Discovery Service (BPN Finder)](https://github.com/eclipse-tractusx/sldt-bpn-discovery) | bpndiscovery: [0.4.2](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/bpndiscovery-0.4.2) | [0.5.2](https://github.com/eclipse-tractusx/sldt-bpn-discovery/releases/tag/v0.5.2) |
| [Eclipse Data Space Connector (Tractus-X EDC)](https://github.com/eclipse-tractusx/tractusx-edc) | [0.8.0](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.8.0) | [0.8.0](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.8.0) |
| [Golden Record Business Partner Number (BPN) Service](https://github.com/eclipse-tractusx/bpdm) | bpdm: [5.2.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-5.2.0) | [6.2.0](https://github.com/eclipse-tractusx/bpdm/releases/tag/v6.2.0) |
| [Identity and Access Management (IAM)](https://github.com/eclipse-tractusx/portal-iam) | centralidp: [4.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/centralidp-4.0.0)<br/>sharedidp: [4.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/sharedidp-4.0.0) | keycloak: [25.0.6](https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-25-0-0) |
| [Industry Core KIT](https://eclipse-tractusx.github.io/docs-kits/category/industry-core-kit) | n/a | [1.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/Industry%20Core%20Kit/Industry%20Core%20Kit%20Changelog) |
| [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents) | remoting-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/remoting-agent-1.14.24)<br>provisioning-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/provisioning-agent-1.14.24)<br>conforming-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/conforming-agent-1.14.24)<br>matchmaking-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/matchmaking-agent-1.14.24) | remoting-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br>provisioning-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br>conforming-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24)<br>matchmaking-agent: [1.14.24](https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24) |
| [(Knowledge) Agents KIT](https://eclipse-tractusx.github.io/docs-kits/category/agents-kit) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/knowledge-agents/Knowledge%20Agent%20Changelog) |
| [Policy Hub](https://github.com/eclipse-tractusx/policy-hub) | policy-hub: [1.2.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.2.0) | [1.2.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/v1.2.0) |
| [Portal](https://github.com/eclipse-tractusx/portal) | portal: [2.3.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.3.0) | registration: [2.1.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.1.0)<br>frontend: [2.3.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.3.0)<br>backend: [2.3.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.3.0)<br>assets: [2.3.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.3.0) |
| [Self Description (SD) Factory](https://github.com/eclipse-tractusx/sd-factory) | sdfactory: [2.1.22](https://github.com/eclipse-tractusx/sd-factory/releases/tag/sdfactory-2.1.22) | [v2.1.14](https://github.com/eclipse-tractusx/sd-factory/releases/tag/v2.1.14) |
| [Semantic Hub](https://github.com/eclipse-tractusx/sldt-semantic-hub) | semantic-hub: [0.4.2](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/semantic-hub-0.4.2) | [0.5.2](https://github.com/eclipse-tractusx/sldt-semantic-hub/releases/tag/v0.5.2) |
| [SSI Authority & Schema Registry](https://github.com/eclipse-tractusx/ssi-authority-schema-registry) | ssi-asr: [1.1.0](https://github.com/eclipse-tractusx/ssi-authority-schema-registry/releases/tag/ssi-asr-1.1.0) | [1.1.0](https://github.com/eclipse-tractusx/ssi-authority-schema-registry/releases/tag/v1.1.0) |
| [SSI Credential Issuer](https://github.com/eclipse-tractusx/ssi-credential-issuer) | ssi-credential-issuer: [1.2.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/ssi-credential-issuer-1.2.0) | [1.2.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/v1.2.0) |
| [Traceability KIT](https://eclipse-tractusx.github.io/docs-kits/category/traceability-kit) | n/a | [6.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Traceability%20Kit%20Changelog) |

### Known Knowns

- All not listed products and KITs have not been updated and are based on the release 24.08
  - These products were not part of the e2e testing phase
  - And are not tested with the current EDC version 0.8.0
- [BPN DID Resolution Service (BDRS)](https://github.com/eclipse-tractusx/bpn-did-resolution-service) is unchanged but was part of the testing phase
- [PURIS](https://github.com/eclipse-tractusx/puris) is unchanged but regression tests were executed
- [Knowledge Agents](https://github.com/eclipse-tractusx/knowledge-agents) was updated to version 0.8.0 but only tested with the EDC release candidate 0.8.0-RC4
- This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
- For E2E Testing, the [2.1.1](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v2.1.1) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
- Security concern about exposed technical users when subscribing BPDM services over the Portal -> see  [according issue](https://github.com/eclipse-tractusx/sig-release/issues/987)
- BPNL Group Validation for a Policy does not allow a Collection as a rightOperand -> see [Issue](https://github.com/eclipse-tractusx/tractusx-edc/issues/1674)
- EDR token refresh implementation is inconsistent and deviates from the documentation -> see [Issue](https://github.com/eclipse-tractusx/tractusx-edc/issues/1565)
- Consumer-Asset with the same ID as a Provider-Asset Cannot Be Deleted After Contract Negotiation -> see [Issue](https://github.com/eclipse-edc/Connector/issues/4583)
- Termination of pull transfer process from consumer side does not succeed -> see [Issue](https://github.com/eclipse-edc/Connector/issues/4592)
- Fatal error while resume PULL transfer process -> see [Issue](https://github.com/eclipse-edc/Connector/issues/4591)

### Runtime

The release has been tested

- on [Kubernetes](https://en.wikipedia.org/wiki/Kubernetes) versions: `1.28.9`
- with [PostgreSQL](https://en.wikipedia.org/wiki/PostgreSQL) versions: `15.2.1.`

Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x

### Using helm with central helm registry

You can search all released helm charts with the following commands:

```shell
helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
helm repo update tractusx-dev
helm search repo tractusx-dev --versions
```

For installation details, please see the referenced Repositories / Releases.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by ther3sa** [comment]
@ds-hzimmer could you check if the Kubernetes version is correct and add the PostgreSQL version. 
Thank you for your support!

**Comment by ther3sa** [comment]
@lgblaumeiser Is this still correct, or has the Tractus-X EDC already been tested on another Postgresql version?
Thank you for your help!

> Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,79 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+| [Digital Twin Kit](https://eclipse-tractusx.github.io/docs-kits/category/digital-twin-kit) | n/a | [2.0.1](https://eclipse-tractusx.github.io/docs-kits/kits/Digital%20Twin%20Kit/Digital%20Twin%20Kit%20Changelog) |
+[EcoPass Kit](https://eclipse-tractusx.github.io/docs-kits/category/eco-pass-kit) | n/a | [1.5.0](https://eclipse-tractusx.github.io/docs-kits/kits/Eco_Pass_KIT/changelog) |
+| [ESS Kit](https://eclipse-tractusx.github.io/docs-kits/category/ess-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/ESS-Kit/ESS%20Kit%20Changelog) |
+| [Identity and Access Management (IAM)](https://github.com/eclipse-tractusx/portal-iam) | centralidp: [3.0.1](https://github.com/eclipse-tractusx/portal-iam/releases/tag/centralidp-3.0.1)<br/>sharedidp: [3.0.1](https://github.com/eclipse-tractusx/portal-iam/releases/tag/sharedidp-3.0.1) | keycloak: [23.0.7](https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-23-0-7) |
```
Comment (evegufy) [inline CHANGELOG.md:46]: Please move to "Updated", it's part of this QG issue https://github.com/eclipse-tractusx/sig-release/issues/920
```suggestion
| [Identity and Access Management (IAM)](https://github.com/eclipse-tractusx/portal-iam) | centralidp: [4.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/centralidp-4.0.0)<br/>sharedidp: [4.0.0](https://github.com/eclipse-tractusx/portal-iam/releases/tag/sharedidp-4.0.0) | keycloak: [25.0.6](https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-25-0-0) |
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,79 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
...
+| [PCF Exchange Kit](https://eclipse-tractusx.github.io/docs-kits/category/pcf-exchange-kit) | n/a | [1.2.0](https://eclipse-tractusx.github.io/docs-kits/kits/PCF%20Exchange%20Kit/PCF%20Exchange%20Kit%20Changelog) |
+| [PURIS](https://github.com/eclipse-tractusx/puris) | puris: [2.7.0](https://github.com/eclipse-tractusx/puris/releases/tag/puris-2.7.0) | [2.1.0](https://github.com/eclipse-tractusx/puris/releases/tag/2.1.0) |
+| [PURIS Kit](https://eclipse-tractusx.github.io/docs-kits/category/puris-kit) | n/a | [0.3.0](https://eclipse-tractusx.github.io/docs-kits/kits/PURIS%20Kit/PURIS%20Kit%20Changelog) |
+| [Self Description (SD) Factory](https://github.com/eclipse-tractusx/sd-factory) | sdfactory: [2.1.21](https://github.com/eclipse-tractusx/sd-factory/releases/tag/sdfactory-2.1.21) | [v2.1.13](https://github.com/eclipse-tractusx/sd-factory/releases/tag/v2.1.13) |
```
Comment (evegufy) [inline CHANGELOG.md:55]: Please check with testmanagement and product team, I think unchanged and untested is wrong

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by ther3sa** [comment]
> The [known knowns from the previous release](https://github.com/eclipse-tractusx/tractus-x-release/blob/c1242583682d2d9f7d13a7ea1583bfedc2866164/CHANGELOG.md?plain=1#L142) should be checked if they apply still to this release.
> 
> I know the following is still relevant:
> 
> * This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
> 
> As well as the following updated note:
> 
> * For E2E Testing, the [2.1.1](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v2.1.1) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
> 
> @nicoprow the following should be removed, right?
> 
> * BPDM has some Known Knowns regarding Portal integration listed in detail [here](https://github.com/eclipse-tractusx/bpdm/blob/release/6.1.x/docs/architecture/11_Risks_And_Technical_Debts.md)
> 
> The following is still relevant (was heavily discussed in the last months):
> 
> * Security concern about exposed technical users when subscribing BPDM services over the Portal
> 
> but the link to the according sig-release issue could be added [sig-release#987](https://github.com/eclipse-tractusx/sig-release/issues/987)
> 
> Just as a note, the following are solved by now:
> 
> * BPDM standard configuration not compatible with Portal's Central-IDP configuration
> * Portal's Partner Network page does not currently show business partner member data due to an authorization issue
> * Currently not possible to add company's site and address business partners over the Portal due to authorization issue

Thank you @evegufy

**Comment by lgblaumeiser** [comment]
> @lgblaumeiser Is this still correct, or has the Tractus-X EDC already been tested on another Postgresql version? Thank you for your help!
> 
> > Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x

Hi @ther3sa, the released helm chart refers to Postgresql 15.2.1. But internal testing of EDC is done using version 16.4. So I can confirm 16.4. As I would assume, that during end-to-end testing people used 15.2.1 as indicated by the helm charts, I would fo that far that we can expect this statement to be valid

**Comment by ther3sa** [comment]
> > @lgblaumeiser Is this still correct, or has the Tractus-X EDC already been tested on another Postgresql version? Thank you for your help!
> > > Note: Tractus-X EDC has been tested on both Postgresql versions: 15.x and 16.x
> 
> Hi @ther3sa, the released helm chart refers to Postgresql 15.2.1. But internal testing of EDC is done using version 16.4. So I can confirm 16.4. As I would assume, that during end-to-end testing people used 15.2.1 as indicated by the helm charts, I would fo that far that we can expect this statement to be valid

okay thank you :)

**Comment by nicoprow** [comment]
> The [known knowns from the previous release](https://github.com/eclipse-tractusx/tractus-x-release/blob/c1242583682d2d9f7d13a7ea1583bfedc2866164/CHANGELOG.md?plain=1#L142) should be checked if they apply still to this release.
> 
> I know the following is still relevant:
> 
>     * This Tractus-X release does not include a Managed Identity Wallet (MIW). The development team decided on a temporary bridging solution, based on a commercial application, to test and ship Release 24.08. Without implementing this interim solution, participants will not be able to fully utilize this Tractus-X release in consequence.
> 
> 
> As well as the following updated note:
> 
>     * For E2E Testing, the [2.1.1](https://github.com/SAP/ssi-dim-middle-layer/releases/tag/v2.1.1) of the [SSI DIM Middle Layer](https://github.com/SAP/ssi-dim-middle-layer) was used - this component is FOSS but the currently used wallet (see previous item) is not.
> 
> 
> @nicoprow the following should be removed, right?
> 
>     * BPDM has some Known Knowns regarding Portal integration listed in detail [here](https://github.com/eclipse-tractusx/bpdm/blob/release/6.1.x/docs/architecture/11_Risks_And_Technical_Debts.md)
> 
> 
> The following is still relevant (was heavily discussed in the last months):
> 
>     * Security concern about exposed technical users when subscribing BPDM services over the Portal
> 
> 
> but the link to the according sig-release issue could be added [sig-release#987](https://github.com/eclipse-tractusx/sig-release/issues/987)
> 
> Just as a note, the following are solved by now:
> 
>     * BPDM standard configuration not compatible with Portal's Central-IDP configuration
> 
>     * Portal's Partner Network page does not currently show business partner member data due to an authorization issue
> 
>     * Currently not possible to add company's site and address business partners over the Portal due to authorization issue

@evegufy 

Right, the only Known-knowns are the exposed service accounts. The updated documentation of BPDM takes that into account: https://github.com/eclipse-tractusx/bpdm/blob/release/6.2.x/docs/architecture/11_Risks_And_Technical_Debts.md#exposed-technical-users-on-portal

So I would also only keep the point you wrote: "Security concern about exposed technical users when subscribing BPDM services over the Portal"

**Review by evegufy — APPROVED** [review]

**Comment by ther3sa** [comment]
Sanity Check

 ```
Found 11 releases
Extracting tables for release: 24.12
Release consists of 67 lines

Processing table 'Updated'
Checking LINKS for KIT: BPDM Kit
Checking LINKS for KIT: Circularity Kit
Checking LINKS for KIT: DCM Kit
  ERROR: KIT DCM Kit has invalid link 'https://eclipse-tractusx.github.io/docs-kits/category/dcm-kit'
  ERROR: Changelog of KIT DCM Kit has invalid link 'https://eclipse-tractusx.github.io/docs-kits/kits/DCM-Kit/changelog'
Checking LINKS for FOSS Component: Digital Twin Registry
Checking CHART for FOSS Component: Digital Twin Registry
  ERROR: Chart version of FOSS Chart digital-twin-registry has version 0.6.2 while version 0.6.3 is available in tractusx-dev
Checking LINKS for FOSS Component: Discovery Finder
Checking CHART for FOSS Component: Discovery Finder
Checking LINKS for FOSS Component: Discovery Service (BPN Finder)
Checking CHART for FOSS Component: Discovery Service (BPN Finder)
Checking LINKS for FOSS Component: Eclipse Data Space Connector (Tractus-X EDC)
  ERROR: Release version '0.8.0' and link version 'https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.8.0-rc6' of FOSS Chart [0.8.0](https do not match
  ERROR: Release version '0.8.0' and link version 'https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.8.0-rc6' of FOSS APP Eclipse Data Space Connector (Tractus-X EDC) do not match
Checking CHART for FOSS Component: Eclipse Data Space Connector (Tractus-X EDC)
  ERROR: Could not retrieve latest Helm chart or app version for [0.8.0](https.
  ERROR: Chart version of FOSS Chart [0.8.0](https has version 0.8.0 while version None is available in tractusx-dev
  ERROR: App version of FOSS Chart [0.8.0](https (app Eclipse Data Space Connector (Tractus-X EDC)) has app version 0.8.0 while version None is set in Chart available in tractusx-dev
Checking LINKS for FOSS Component: Golden Record Business Partner Number (BPN) Service
Checking CHART for FOSS Component: Golden Record Business Partner Number (BPN) Service
Checking LINKS for FOSS Component: Identity and Access Management (IAM)
  ERROR: Release version '25.0.6' and link version 'https://www.keycloak.org/docs/latest/release_notes/index.html#keycloak-25-0-0' of FOSS APP Identity and Access Management (IAM) do not match
Checking CHART for FOSS Component: Identity and Access Management (IAM)
Checking LINKS for KIT: Industry Core KIT
Checking LINKS for FOSS Component: Knowledge Agents
  ERROR: Release version '1.14.24' and link version 'https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/matchmaking-agent-1.14.24-SNAPSHOT' of FOSS Chart remoting-agent do not match
  ERROR: Release of FOSS APP Knowledge Agents has invalid link 'https://github.com/eclipse-tractusx/knowledge-agents/releases/tag/v1.14.24'
Checking CHART for FOSS Component: Knowledge Agents
  ERROR: Chart version of FOSS Chart remoting-agent has version 1.14.24 while version 1.13.22 is available in tractusx-dev
  ERROR: App version of FOSS Chart remoting-agent (app Knowledge Agents) has app version 1.14.24 while version 1.13.22 is set in Chart available in tractusx-dev
Checking LINKS for KIT: (Knowledge) Agents KIT
Checking LINKS for FOSS Component: Policy Hub
Checking CHART for FOSS Component: Policy Hub
Checking LINKS for FOSS Component: Portal
Checking CHART for FOSS Component: Portal
  ERROR: App version of FOSS Chart portal (app Portal) has app version 2.1.0 while version 2.3.0 is set in Chart available in tractusx-dev
Checking LINKS for FOSS Component: Self Description (SD) Factory
Checking CHART for FOSS Component: Self Description (SD) Factory
Checking LINKS for FOSS Component: Semantic Hub
Checking CHART for FOSS Component: Semantic Hub
Checking LINKS for FOSS Component: SSI Authority & Schema Registry
Checking CHART for FOSS Component: SSI Authority & Schema Registry
Checking LINKS for FOSS Component: SSI Credential Issuer
Checking CHART for FOSS Component: SSI Credential Issuer
Checking LINKS for KIT: Traceability Kit
```

**Review by almadigabor — APPROVED** [review]

**Review by stephanbcbauer — APPROVED** [review]
LGTM

**Pull request merged**